client/js/components/statement/splitStatement/SideBar.vue

```diff
@@ -155,6 +155,7 @@
 
     <dp-button-row
       class="p-2"
+      data-cy="assignedTags"
```

Comment:
i think that the kind of action should be part of the hook name, like `saveAssignedTags`, what do you think?

Reply:
DpButtonRow component in demosplan-ui has 2 buttons anyway, with:
```
:data-cy="`${dataCy}:saveButton`" 
:data-cy="`${dataCy}:abortButton`"
```
then here with my changes:
`data-cy="assignedTags:saveButton"` and `data-cy="assignedTags:abortButton"`

Reply:
@spiess-demos

Reply:
ok, i did not realize that. thanks for the explanation!